### PR TITLE
Bug/686

### PR DESCRIPTION
**Pull request by LeandroGuillen**
Convenience operation GET /v1/contextTypes now returns all types even if they have no attributes.

This fixes bug https://github.com/telefonicaid/fiware-orion/issues/686.

### CODE REVIEW

File: src/app/contextBroker/CMakeLists.txt [hunk 1]
```diff
@@ -80,6 +80,9 @@ ELSEIF(${DISTRO} STREQUAL "Ubuntu_14.04.1_LTS")
   # FIXME P4: Issue #476: remove or keep?
```
Comment (fgalan) [inline src/app/contextBroker/CMakeLists.txt]: Remember to include the entry corresponding to the fix y CHANGES_NEXT_RELEASE

File: src/lib/mongoBackend/mongoQueryTypes.cpp [hunk 2]
```diff
@@ -48,7 +48,7 @@ HttpStatusCode mongoEntityTypes
   unsigned int limit          = atoi(uriParams[URI_PARAM_PAGINATION_LIMIT].c_str());
   std::string  detailsString  = uriParams[URI_PARAM_PAGINATION_DETAILS];
   bool         details        = (strcasecmp("on", detailsString.c_str()) == 0)? true : false;
-
+  //bool         collapse       = uriParams[URI_PARAM_COLLAPSE] == "true";
```
Comment (fgalan) [inline src/lib/mongoBackend/mongoQueryTypes.cpp]: Leftover comment?

File: src/app/contextBroker/CMakeLists.txt (hunk 1 above)
Comment (LeandroGuillen) [inline src/app/contextBroker/CMakeLists.txt]: Fixed in  0143b6d

File: src/lib/mongoBackend/mongoQueryTypes.cpp (hunk 2 above)
Comment (LeandroGuillen) [inline src/lib/mongoBackend/mongoQueryTypes.cpp]: Fixed in  0143b6d

**Comment by fgalan** [comment]
LGTM

**Pull request merged**